src/explore-education-statistics-admin/src/pages/release/edit-release/content/ReleaseContext.tsx

```diff
@@ -199,6 +199,46 @@ export const releaseReducer: Reducer<
     default: {
       return draft;
     }
+    case 'UPDATE_BLOCK_COMMENTS': {
```

Comment:
Would it be neater to put the "default" case at the end of the switch statement?

Reply:
Yep indeed. Bad form by Mark